examples/getting_started/pt/src/cifar10_fl.py

```diff
@@ -122,6 +122,7 @@ def evaluate(input_weights):
 
         # (6) evaluate on received model for model selection
         accuracy = evaluate(input_model.params)
+        summary_writer.add_scalar(tag="global_model_accuracy", scalar=accuracy, global_step=input_model.current_round)
```

Comment:
suggest move this change to another PR.

Reply:
Just removed the change, thanks for spotting this out!